src/wolff_client.erl

```diff
@@ -103,6 +106,19 @@ check_connectivity(Hosts, ConnConfig) ->
             {error, tr_reasons(Reasons)}
     end.
 
+-spec check_connectivity([host()], kpro:conn_config(), topic()) ->
+        ok | {error, unknown_topic_or_partition | [#{host := binary(), reason := term()}] | any()}.
+check_connectivity(Hosts, ConnConfig, Topic) ->
+  case get_metadata(Hosts, ConnConfig, Topic) of
```

Comment:
`check_connectivity/2` and `check_connectivity/3` seem to do different things:
- the arity 2 version goal seems to be to test only the connection;
- the arity 3 version goal seems to be to check the existence of a topic.  This can fail due to connectivity issues, but that seems secondary for this purpose.

Perhaps it would be more appropriate to call the arity 3 fn something like `check_if_topic_exists`.  WDYT?

Reply:
I agree =)